src/Literate.jl

```diff
@@ -782,16 +782,55 @@ function execute_block(sb::Module, block::String; inputfile::String="<unknown>")
     end
     popdisplay(disp) # IOCapture.capture has a try-catch so should always end up here
     if c.error
-        error("""
-             $(sprint(showerror, c.value))
-             when executing the following code block in file `$(Base.contractuser(inputfile))`
-
-             ```julia
-             $block
-             ```
-             """)
+        # if c.value isa LoadError
+        #     @info "..." c.value.file c.value.line
+        #     bt = remove_common_backtrace(c.backtrace, backtrace())
+        #     st = stacktrace(bt)
+        #     @assert length(bt) >= length(st)
+        #     idx = findlast(sf -> sf.func === :eval, st)
+        #     display(st)
+        #     @show idx st[idx]
+        #     bt_ptr = Ptr{Nothing}(st[idx-1].pointer)
+        #     idx = findfirst(isequal(bt_ptr), bt)
+        #     @show idx
+        #     display(stacktrace(bt[1:idx]))
+        #     throw(EvalException(Base.contractuser(inputfile), block, c.value.error, bt[1:idx]))
+        # else
+        #     throw(EvalException(Base.contractuser(inputfile), block, c.value, remove_common_backtrace(c.backtrace, backtrace())))
+        # end
```

Comment:
This was a naive attempt the get rid of the stackframes introduced by `include_string`, but it didn't quite work right.